source/PlayerInfo.cpp

```diff
@@ -2697,6 +2697,20 @@ void PlayerInfo::RegisterDerivedConditions()
 	flagshipModelProvider.SetHasFunction(flagshipModelFun);
 	flagshipModelProvider.SetGetFunction(flagshipModelFun);
 
+	auto &&outfitTagProvider = conditions.GetProviderPrefixed("outfit tag: ");
+	auto outfitTagFun = [this](const string &name) -> bool
+	{
+		if(!flagship)
```

Comment:
I think you should consider all ships and do different conditions like it was recently done by Amazinite
could be in a follow up PR though

Reply:
hmmm... I will see

Reply:
Basically a flagship/cargo/etc only option would be nice, but its best left for later prob

Reply:
I would like to leave this to a later PR